### PR TITLE
Precision on events max KB size

### PR DESCRIPTION
**Pull request by ngouy**
### Proposed changes

A small precision on the doc that can save developers and support precious minutes

### Merge timing
When should this get merged/published?
- could be ASAP once approved (I guess)?

### CODE REVIEW

**Comment by ngouy** [comment]
source: A supoprt request from our team at Capable
https://github.com/capable-health

image

File: src/connections/sources/catalog/libraries/server/ruby/index.md [hunk 1]
```diff
@@ -413,6 +413,8 @@ By default, our library will flush:
 + whenever messages are queued and there is no outstanding request
 
 There is a maximum of `500KB` per batch request and `32KB` per call.
+> Note that even if batch request accept a maximum of `500KB`, each indivduall items should stay under `32KB`.
```
Comment (ngouy) [inline src/connections/sources/catalog/libraries/server/ruby/index.md:416]:
```suggestion
> Note that even if batch request accepts a maximum of `500KB`, each individual item should stay under `32KB`.
```

**Comment by markzegarelli** [comment]
Hi @ngouy! Thanks for submitting this update. I chatted with an engineer on our side, and this change impacts all sources, not just the Ruby source. 

I will close this PR, make the broader update in a separate, non-forked PR, and add you as a contributor that.

**Comment by ngouy** [comment]
🙏 thank you sir

**Pull request closed without merging**